src/test/resources/hudson/views/configuration-as-code.yaml

```diff
@@ -0,0 +1,25 @@
+jenkins:
+  views:
+    - all:
+        name: "all"
+    - list:
+        columns:
+          - "status"
+          - "weather"
+          - "jobName"
+          - "lastSuccess"
+          - "lastFailure"
+          - "lastDuration"
+          - "buildButton"
+        description: "test view"
+        includeRegex: ".*"
+        jobFilters:
+          - buildDurationFilter:
+              amount: 24.0F
+              amountTypeString: "Hours"
+              buildCountTypeString: "Latest"
+              buildDurationMinutes: "10"
+              includeExcludeTypeString: "includeMatched"
```

Comment:
Unfortunate that we have property names ending with `String`, and a numeric field typed as a string.